unlock-app/src/stories/pages.stories.js

```diff
@@ -1,20 +1,22 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
-import Pages from '../components/pages'
+import Home from '../pages'
+import About from '../pages/about'
+import Jobs from '../pages/jobs'
 
-storiesOf('Content pages', Pages)
+storiesOf('Content pages', module)
```

Comment:
This means auto-reload won't work, right?

Reply:
Nope works fine for me.